From 0825bddef65c11928ae6d37a2a7702e2d9435143 Mon Sep 17 00:00:00 2001
From: Dirk Mueller <dirk@dmllr.de>
Date: Sun, 1 Mar 2020 20:54:43 +0100
Subject: [PATCH] Fix syntax error in image widths

I get when building sahara this error:

  File "/usr/lib/python3.6/site-packages/sphinx/writers/html5.py", line 548, in visit_image
     atts['width'] = int(atts['width']) * scale
  ValueError: invalid literal for int() with base 10: '800px'

As it turns out, the default is pixels anyway, so we don't need to add a
type suffix. For scale to work, we need to specify both width and height
for images, otherwise sphinx reports an error.

Change-Id: If0f8383e37a8b1185a8a08b24c79b313fdcb498c
---
 doc/source/intro/architecture.rst | 5 +++--
 doc/source/intro/overview.rst     | 5 +++--
 2 files changed, 6 insertions(+), 4 deletions(-)

diff --git a/doc/source/intro/architecture.rst b/doc/source/intro/architecture.rst
index 608bede3..6ebd2c44 100644
--- a/doc/source/intro/architecture.rst
+++ b/doc/source/intro/architecture.rst
@@ -2,8 +2,9 @@ Architecture
 ============
 
 .. image:: ../images/sahara-architecture.svg
-    :width: 800 px
-    :scale: 100 %
+    :width: 960
+    :height: 635
+    :scale: 83 %
     :align: left
 
 
diff --git a/doc/source/intro/overview.rst b/doc/source/intro/overview.rst
index 1560b417..6f121541 100644
--- a/doc/source/intro/overview.rst
+++ b/doc/source/intro/overview.rst
@@ -75,8 +75,9 @@ The sahara product communicates with the following OpenStack services:
   like passwords and private keys in a secure storage.
 
 .. image:: ../images/openstack-interop.png
-    :width: 800 px
-    :scale: 99 %
+    :width: 960
+    :height: 720
+    :scale: 83 %
     :align: left
 
 General Workflow
-- 
2.25.1

